src/frontend/efiling-demo/src/components/page/error/Error.js

```diff
@@ -1,9 +1,17 @@
 import React from "react";
 import PropTypes from "prop-types";
+import queryString from "query-string";
+import { useLocation } from "react-router-dom";
 import { Header, Footer, Button } from "shared-components";
 import { propTypes } from "../../../types/propTypes";
 
 export default function Error({ page: { header, error } }) {
+  const location = useLocation();
+  const queryParams = queryString.parse(location.search);
+
+  const status = queryParams.status;
+  const message = queryParams.message;
```

Comment:
the way this is being done needs to be discussed, the way I had set it up earlier is that the querystring gets parsed at a HOC and then passed in as a prop to this component.

If you'd like to use it this way, then we should remove the code in the HOC to parse the error and pass it into this component

Reply:
No you right, I missed what it was doing there when I was looking at it yesterday.